internal/lightcone/abundance/warmthshortenscoldnights/warmthshortenscoldnights.go

```diff
@@ -0,0 +1,67 @@
+package warmthshortenscoldnights
+
+import (
+	"github.com/simimpact/srsim/pkg/engine"
+	"github.com/simimpact/srsim/pkg/engine/equip/lightcone"
+	"github.com/simimpact/srsim/pkg/engine/event"
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/engine/prop"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	WarmthShortensColdNightsHPBuff     key.Modifier = "warmth_shortens_cold_nights_hp_buff"
+	WarmthShortensColdNightsHealCheck  key.Modifier = "warmth_shortens_cold_nights_heal_check"
+)
+
+func init() {
+	lightcone.Register(key.WarmthShortensColdNights, lightcone.Config{
+		CreatePassive: Create,
+		Rarity:        4,
+		Path:          model.Path_ABUNDANCE,
+		Promotions:    promotions,
+	})
+
+	//hp buff goes here
+	modifier.Register(WarmthShortensColdNightsHPBuff, modifier.Config{
+		StatusType: model.StatusType_STATUS_BUFF,
+	})
+	//Check if action is basic atk / skill
+	modifier.Register(WarmthShortensColdNightsHealCheck, modifier.Config{
+		Listeners: modifier.Listeners{
+			OnAfterAction: healTeamOnBasicOrSkill,
+		},
+	})
+}
+
+func Create(engine engine.Engine, owner key.TargetID, lc info.LightCone) {
+	amt := 0.12 + 0.04*float64(lc.Imposition)
+	//OnStart :
+	engine.AddModifier(owner, info.Modifier{
+		Name:   WarmthShortensColdNightsHPBuff,
+		Source: owner,
+		Stats:  info.PropMap{prop.HPPercent: amt},
+		State:  amt,
+	})
+	
+	engine.AddModifier(owner, info.Modifier{ 
+		Name:   WarmthShortensColdNightsHealCheck,
+		Source: owner,
+		State:  0.02 + 0.005 * float64(lc.Imposition),
+	})
```

Comment:
this should be simplified to a single modifier & add modifier call. The first call has the right idea:
1. Create a `Stats` line setting the `HPPercent` prop (will always have this HP increase as long as modifier is attached, which is always)
2. Set `State` to be used in the `healTeamOnBasicOrSkill` for the heal

Note: the heal amount should be `0.015 + 0.005 * float64(lc.Imposition)`